src/main/cli/settings.c

```diff
@@ -293,16 +293,18 @@ static const char * const lookupTablePwmProtocol[] = {
 
 static const char * const lookupTableLowpassType[] = {
     "PT1",
-    "BIQUAD",
+    "BUTTERWORTH",
```

Comment:
Convert it into a name in mathematics is nice, but I think it should be written in related documents and explain to users why 'biquad is 'gone''?

Reply:
'Butterworth' is a generic name for a whole range of filters.  Our PT1, PT2 and PT3 filters are all butterworth filters, I think, just different order.
Our 'Biquad' filter is a historical name that people understand and know well.
My personal feeling is that we shouldn't change its name, even though technically the current 'Biquad' name is definitely not 'correct', and I would like to change it, there is too much history I think to make such a change worthwhile.

Reply:
- **Bessel** filters are optimized for **maximally linear phase response** in the passband, which means _filtered signals won't get their shape distorted in the passband_.
- **Butterworth** filters are optimized for **maximally flat frequency response** in the passband (at the expense of vayring phase delay).

Filters with varying phase delay will induce some level of **signal distortion** (PT2/PT3 and especially our Butterworth biquad, I don't know about the PT1).

**Biquad** is a name for all second order filters, for example: 
- Bessel
- our "BIQUAD" (2nd order Butterworth) 
- our PT2
- ...
- 
I made this PR just in case Bessel filters are coincidentally the ideal filters for the P-term and D-term. So I just want to make sure we are carefully looking into it before dismissing a chance. Right now this PR is more of a "test site".

If this is really going anywhere, I will take your semantic changes into account. I can see how leaving its name as "BIQUAD" would be beneficial.